src/main/java/frc/robot/Climb.java

```diff
@@ -21,7 +26,15 @@ public void runWinch(double power) {
     }
 
     public void moveArms(double power) {
-        armMotor.set(power);
+        armMotor.set(-power);
+    }
+
+    public void lock(boolean on) {
```

Comment:
Should the logic for the not moving the winch when the lock is engaged be in the Climb class?
Should this be part of this pull request or should an issue be made for this?